### PR TITLE
bitmart - watchTicker & watchTickers reorg using MultiSubscriptions

### PR DESCRIPTION
**Pull request by ttodua**
```
npm run cli.js -- bitmart watchTickers  "['BTC/USDT','ETH/USDT']"

> ccxt@4.2.49 cli.js
> node ./examples/js/cli.js bitmart watchTickers ['BTC/USDT','ETH/USDT']

2024-02-23T10:34:31.575Z
Node.js: v20.10.0
CCXT v4.2.49
bitmart.watchTickers (BTC/USDT,ETH/USDT)
{
  'BTC/USDT': {
    symbol: 'BTC/USDT',
    timestamp: 1708684477000,
    datetime: '2024-02-23T10:34:37.000Z',
    high: 52062.49,
    low: 50831.11,
    open: 51844.57,
    close: 51101.18,
    last: 51101.18,
    change: -743.39,
    percentage: -1.4338820825401772,
    average: 51472.875,
    quoteVolume: 429705105.06,
    info: {
      base_volume_24h: '429705105.06',
      high_24h: '52062.49',
      last_price: '51101.18',
      low_24h: '50831.11',
      open_24h: '51844.57',
      s_t: 1708684477,
      symbol: 'BTC_USDT'
    }
  }
}
{
  'ETH/USDT': {
    symbol: 'ETH/USDT',
    timestamp: 1708684477000,
    datetime: '2024-02-23T10:34:37.000Z',
    high: 3036.01,
    low: 2913.3,
    open: 3010.61,
    close: 2940.38,
    last: 2940.38,
    change: -70.23,
    percentage: -2.332749841394269,
    average: 2975.495,
    quoteVolume: 263454277.55,
    info: {
      base_volume_24h: '263454277.55',
      high_24h: '3036.01',
      last_price: '2940.38',
      low_24h: '2913.30',
      open_24h: '3010.61',
      s_t: 1708684477,
      symbol: 'ETH_USDT'
    }
  }
}
```

similar with `BTC/USDT:USDT ... ` swap symbols.

singular watchTicker:
```
npm run cli.js -- bitmart watchTicker BTC/USDT:USDT    

> ccxt@4.2.49 cli.js
> node ./examples/js/cli.js bitmart watchTicker BTC/USDT:USDT

2024-02-23T10:38:22.005Z
Node.js: v20.10.0
CCXT v4.2.49
bitmart.watchTicker (BTC/USDT:USDT)
{
  symbol: 'BTC/USDT:USDT',
  bid: 51170.8,
  bidVolume: 11653,
  ask: 51171.3,
  askVolume: 12190,
  close: 51171.1,
  last: 51171.1,
  average: 51168.5,
  quoteVolume: 51344191.637100644,
  info: {
    symbol: 'BTCUSDT',
    last_price: '51171.1',
    volume_24: '51344191.637100645833040206',
    range: '-0.0126471643726196127',
    fair_price: '51168.5',
    ask_price: '51171.3',
    ask_vol: '12190',
    bid_price: '51170.8',
    bid_vol: '11653'
  }
}
```

### CODE REVIEW

**Comment by ttodua** [comment]
depends on .reverse PRs

File: ts/src/pro/bitmart.ts [hunk 1]
```diff
@@ -867,6 +840,10 @@ export default class bitmart extends bitmartRest {
         if (data === undefined) {
             return;
         }
+        const isSwap = ('group' in message);
+        if (isSwap) {
+            data.reverse (); // fix chronological order
```
Comment (carlosmiei) [inline ts/src/pro/bitmart.ts:845]: again I'm not a big fan of this reverse, when we can easily iterate backwards without iterating twice over the array

Comment (ttodua) [reply ts/src/pro/bitmart.ts]: ah yes, this update was before you gave me correct idea about reversing. now fixed ✌️

**Comment by carlosmiei** [comment]
DEMO

```
Python v3.11.7
CCXT v4.2.58
bitmart.watchTradesForSymbols(['BTC/USDT', 'ETH/USDT'])
[{'amount': 0.00067,
  'cost': 43.5158434,
  'datetime': '2024-03-04T11:33:40.000Z',
  'fee': None,
  'fees': [],
  'id': None,
  'info': {'ms_t': 1709552020997,
           'price': '64949.02',
           's_t': 1709552020,
           'side': 'sell',
           'size': '0.00067',
           'symbol': 'BTC_USDT'},
  'order': None,
  'price': 64949.02,
  'side': 'sell',
  'symbol': 'BTC/USDT',
  'takerOrMaker': None,
  'timestamp': 1709552020000,
  'type': None},
 {'amount': 0.00032,
```
```
Python v3.11.7
CCXT v4.2.58
bitmart.watchOrderBookForSymbols(['BTC/USDT', 'ETH/USDT'])
{'asks': [[64977.33, 0.01091], [64977.34, 0.00777], [64979.01, 0.21254], [64979.4, 0.13723], [64979.79, 0.05122], [64980.18, 0.15744], [64980.57, 0.0539], [64980.96, 0.11303], [64981.35, 0.07314], [64981.74, 0.16476], [64982.33, 0.1825], [64982.72, 0.18989], [64983.11, 0.16794], [64983.5, 0.1282], [64983.89, 0.09608], [64984.28, 0.18163], [64984.67, 0.1468], [64985.06, 0.21715], [64985.64, 0.06259], [64986.03, 0.15365], [64986.42, 0.14033], [64986.81, 0.10646], [64987.2, 0.06438], [64987.59, 0.13249], [64987.98, 0.08702], [64988.37, 0.18811], [64988.76, 0.07915], [64989.15, 0.16241], [64989.29, 0.16161], [64989.54, 0.14536], [64989.93, 0.2185], [64990.32, 0.10103], [64990.71, 0.08744], [64991.1, 0.1751], [64991.49, 0.20124], [65002.28, 0.64644], [65009.86, 2.71788], [65023.63, 2.09686], [65036.76, 3.66218], [65049.13, 3.49941], [65063.6, 1.00231], [65064.08, 3.23051], [65071.6, 2.72252], [65081.36, 2.52284], [65082.31, 2.29089], [65092.64, 3.74602], [65099.74, 0.80806], [65105.45, 3.15855], [65118.77, 1.77059], [65129.51, 0.90479], [65130.48, 1.42148], [65142.91, 2.92504], [65156.16, 3.30877], [65168.18, 1.05824], [65179.37, 0.9938], [65190.41, 0.79449], [65191.2, 2.03482], [65201.77, 2.62653], [65213.34, 2.45146], [65224.55, 2.40954], [65230.0, 0.00459], [65234.4, 3.31059], [65234.55, 0.72405], [65246.55, 1.90021], [65257.35, 1.96359], [65267.05, 1.20757], [65276.78, 0.8546], [65289.08, 2.12837], [65299.08, 0.72319], [65300.91, 1.09178], [65312.79, 2.9276], [65326.85, 3.24624], [65339.15, 2.06579], [65351.67, 0.99011], [65362.19, 2.87805], [65368.67, 1.0028], [65371.93, 3.57434], [65379.92, 0.01238], [65381.74, 2.56913], [65438.63, 0.83859], [65450.99, 0.00026], [65512.28, 1.01632], [65572.85, 1.09471], [65609.29, 0.00112], [65625.54, 0.77749], [65683.41, 0.00012], [65688.77, 0.84093], [65691.5, 3.34358], [65743.09, 1.14034], [65806.62, 1.07837], [65821.0, 0.00348], [65870.93, 1.055], [65890.04, 3.07448], [65900.0, 0.2494], [65946.73, 0.74677], [66000.0, 0.05431], [66003.7, 0.0004], [66038.66, 1.06594], [66137.78, 0.79662], [66379.02, 0.98829]],
 'bids': [[64977.32, 0.00142], [64977.31, 0.00273], [64975.64, 0.12124], [64975.25, 0.18896], [64974.86, 0.13267], [64974.47, 0.12807], [64974.08, 0.16023], [64973.69, 0.10346], [64973.3, 0.11695], [64972.91, 0.05918], [64972.52, 0.09198], [64972.13, 0.05536], [64971.74, 0.11059], [64971.35, 0.10849], [64970.96, 0.08252], [64970.57, 0.13131], [64970.18, 0.17492], [64969.74, 0.01939], [64969.6, 0.20924], [64969.21, 0.18632], [64968.82, 0.07539], [64968.43, 0.06721], [64968.04, 0.21228], [64967.65, 0.12439], [64967.26, 0.10558], [64966.87, 0.13169], [64966.28, 0.17826], [64965.89, 0.14435], [64965.5, 0.09208], [64965.11, 0.0976], [64964.72, 0.15501], [64964.33, 0.15173], [64963.94, 0.17183], [64963.55, 0.12717], [64963.16, 0.0575], [64950.32, 0.16161], [64943.28, 2.61963], [64937.33, 0.64644], [64932.83, 3.44326], [64921.23, 0.9137], [64910.71, 2.31508], [64900.43, 2.91704], [64897.0, 0.00026], [64889.47, 2.08359], [64885.16, 0.93718], [64879.73, 3.27611], [64869.29, 0.0034], [64869.28, 2.63588], [64859.55, 2.19285], [64848.75, 1.70917], [64842.81, 1.00376], [64839.87, 0.80806], [64837.65, 1.0014], [64826.5, 3.34688], [64809.72, 3.70686], [64799.0, 0.00251], [64798.12, 1.51548], [64794.47, 2.13972], [64784.74, 3.67816], [64775.01, 3.58262], [64761.23, 1.77896], [64746.19, 2.49677], [64731.09, 2.87799], [64729.4, 0.89492], [64723.02, 1.31278], [64713.26, 3.47127], [64700.91, 3.33475], [64689.28, 2.53177], [64674.95, 3.49183], [64664.84, 3.58156], [64660.09, 1.09298], [64653.05, 1.1081], [64644.7, 1.61844], [64634.95, 3.71415], [64626.16, 0.72081], [64619.72, 3.04355], [64607.81, 3.46621], [64594.77, 1.15215], [64582.6, 1.36349], [64555.64, 0.76063], [64471.51, 0.88165], [64409.48, 0.93766], [64378.21, 0.85934], [64332.02, 1.00653], [64266.58, 0.95757], [64226.76, 1.24], [64161.35, 0.00026], [64148.85, 0.88777], [64091.4, 0.70254], [64076.4, 0.81822], [64004.69, 1.03994], [64000.17, 1.56957], [63953.52, 0.70378], [63884.55, 1.09995], [63802.47, 1.50372], [63789.0, 0.00193], [63783.83, 1.17067], [63747.14, 1.10126], [63620.05, 1.03876], [63421.39, 1.13637]],
 'datetime': '2024-03-04T11:34:28.634Z',
 'nonce': None,
 'symbol': 'BTC/USDT',
 'timestamp': 1709552068634}
{'asks': [[64977.33, 0.0055], [64977.34, 0.00777], [64979.01, 0.21254], [64979.4, 0.13723], [64979.79, 0.05122], [64980.18, 0.15744], [64980.57, 0.0539], [64980.96, 0.11303], [64981.35, 0.07314], [64981.74, 0.16476], [64982.33, 0.1825], [64982.72, 0.18989], [64983.11, 0.16794], [64983.5, 0.1282], [64983.89, 0.09608], [64984.28, 0.18163], [64984.67, 0.1468], [64985.06, 0.21715], [64985.64, 0.06259], [64986.03, 0.15365], [64986.42, 0.14033], [64986.81, 0.10646], [64987.2, 0.06438], [64987.59, 0.13249], [64987.98, 0.08702], [64988.37, 0.18811], [64988.76, 0.07915], [64989.15, 0.16241], [64989.29, 0.16161], [64989.54, 0.14536], [64989.93, 0.2185], [64990.32, 0.10103], [64990.71, 0.08744], [64991.1, 0.1751], [64991.49, 0.20124], [65002.28, 0.64644], [65009.86, 2.71788], [65023.63, 2.09686], [65036.76, 3.66218], [65049.13, 3.49941], [65063.6, 1.00231], [65064.08, 3.23051], [65071.6, 2.72252], [65081.36, 2.52284], [65082.31, 2.29089], [65092.64, 3.74602], [65099.74, 0.80806], [65105.45, 3.15855], [65118.77, 1.77059], [65129.51, 0.90479], [65130.48, 1.42148], [65142.91, 2.92504], [65156.16, 3.30877], [65168.18, 1.05824], [65179.37, 0.9938], [65190.41, 0.79449], [65191.2, 2.03482], [65201.77, 2.62653], [65213.34, 2.45146], [65224.55, 2.40954], [65230.0, 0.00459], [65234.4, 3.31059], [65234.55, 0.72405], [65246.55, 1.90021], [65257.35, 1.96359], [65267.05, 1.20757], [65276.78, 0.8546], [65289.08, 2.12837], [65299.08, 0.72319], [65300.91, 1.09178], [65312.79, 2.9276], [65326.85, 3.24624], [65339.15, 2.06579], [65351.67, 0.99011], [65362.19, 2.87805], [65368.67, 1.0028], [65371.93, 3.57434], [65379.92, 0.01238], [65381.74, 2.56913], [65438.63, 0.83859], [65450.99, 0.00026], [65512.28, 1.01632], [65572.85, 1.09471], [65609.29, 0.00112], [65625.54, 0.77749], [65683.41, 0.00012], [65688.77, 0.84093], [65691.5, 3.34358], [65743.09, 1.14034], [65806.62, 1.07837], [65821.0, 0.00348], [65870.93, 1.055], [65890.04, 3.07448], [65900.0, 0.2494], [65946.73, 0.74677], [66000.0, 0.05431], [66003.7, 0.0004], [66038.66, 1.06594], [66137.78, 0.79662], [66379.02, 0.98829]],
 'bids': [[64977.32, 0.00142], [64977.31, 0.00273], [64975.64, 0.12124], [64975.25, 0.18896], [64974.86, 0.13267], [64974.47, 0.12807], [64974.08, 0.16023], [64973.69, 0.10346], [64973.3, 0.11695], [64972.91, 0.05918], [64972.52, 0.09198], [64972.13, 0.05536], [64971.74, 0.11059], [64971.35, 0.10849], [64970.96, 0.08252], [64970.57, 0.13131], [64970.18, 0.17492], [64969.74, 0.01939], [64969.6, 0.20924], [64969.21, 0.18632], [64968.82, 0.07539], [64968.43, 0.06721], [64968.04, 0.21228], [64967.65, 0.12439], [64967.26, 0.10558], [64966.87, 0.13169], [64966.28, 0.17826], [64965.89, 0.14435], [64965.5, 0.09208], [64965.11, 0.0976], [64964.72, 0.15501], [64964.33, 0.15173], [64963.94, 0.17183], [64963.55, 0.12717], [64963.16, 0.0575], [64950.32, 0.16161], [64943.28, 2.61963], [64937.33, 0.64644], [64932.83, 3.44326], [64921.23, 0.9137], [64910.71, 2.31508], [64900.43, 2.91704], [64897.0, 0.00026], [64889.47, 2.08359], [64885.16, 0.93718], [64879.73, 3.27611], [64869.29, 0.0034], [64869.28, 2.63588], [64859.55, 2.19285], [64848.75, 1.70917], [64842.81, 1.00376], [64839.87, 0.80806], [64837.65, 1.0014], [64826.5, 3.34688], [64809.72, 3.70686], [64799.0, 0.00251], [64798.12, 1.51548], [64794.47, 2.13972], [64784.74, 3.67816], [64775.01, 3.58262], [64761.23, 1.77896], [64746.19, 2.49677], [64731.09, 2.87799], [64729.4, 0.89492], [64723.02, 1.31278], [64713.26, 3.47127], [64700.91, 3.33475], [64689.28, 2.53177], [64674.95, 3.49183], [64664.84, 3.58156], [64660.09, 1.09298], [64653.05, 1.1081], [64644.7, 1.61844], [64634.95, 3.71415], [64626.16, 0.72081], [64619.72, 3.04355], [64607.81, 3.46621], [64594.77, 1.15215], [64582.6, 1.36349], [64555.64, 0.76063], [64471.51, 0.88165], [64409.48, 0.93766], [64378.21, 0.85934], [64332.02, 1.00653], [64266.58, 0.95757], [64226.76, 1.24], [64161.35, 0.00026], [64148.85, 0.88777], [64091.4, 0.70254], [64076.4, 0.81822], [64004.69, 1.03994], [64000.17, 1.56957], [63953.52, 0.70378], [63884.55, 1.09995], [63802.47, 1.50372], [63789.0, 0.00193], [63783.83, 1.17067], [63747.14, 1.10126], [63620.05, 1.03876], [63421.39, 1.13637]],
 'datetime': '2024-03-04T11:34:28.642Z',
 'nonce': None,
 'symbol': 'BTC/USDT',
 'timestamp': 1709552068642}
{'asks': [[3498.6, 0.00958], [3498.61, 0.03379], [3498.69, 2.07037], [3498.75, 2.39453], [3498.8, 2.5456], [3498.83, 2.66888], [3498.87, 0.7998], [3498.9, 0.85495], [3498.94, 2.63068], [3498.97, 2.74891], [3499.03, 1.85356], [3499.06, 1.88958], [3499.1, 1.17156], [3499.15, 0.88579], [3499.18, 1.03261], [3499.2, 2.18459], [3499.24, 2.52451], [3499.27, 2.60104], [3499.32, 2.78785], [3499.35, 1.15587], [3499.39, 0.8751], [3499.42, 0.9731], [3499.46, 1.42857], [3499.5, 2.97997], [3499.54, 0.87038], [3499.57, 2.26651], [3499.61, 1.48082], [3499.64, 1.54484], [3500.0, 0.00154], [3500.34, 3.0012], [3500.51, 29.72567], [3500.94, 31.06504], [3501.8, 31.08638], [3502.33, 64.41507], [3502.9, 43.71048], [3503.45, 45.35982], [3504.02, 54.92622], [3504.67, 47.80557], [3505.48, 56.70803], [3505.59, 12.00482], [3506.11, 54.96626], [3506.83, 67.34527], [3507.46, 42.91909], [3508.16, 22.11191], [3508.72, 38.66972], [3509.27, 56.35677], [3509.83, 20.67794], [3510.48, 40.34268], [3510.59, 60.48656], [3511.1, 25.45966], [3512.0, 34.38476], [3512.53, 59.67524], [3512.58, 15.00603], [3513.05, 25.48052], [3513.83, 34.56468], [3514.31, 27.42947], [3514.79, 29.68947], [3515.32, 47.53059], [3515.84, 59.96467], [3516.65, 66.76271], [3517.22, 46.2545], [3517.8, 49.21817], [3518.66, 64.12218], [3519.38, 45.66986], [3520.09, 52.90242], [3520.67, 61.8717], [3531.87, 0.00474], [3537.64, 21.02845], [3538.4, 3.20271], [3548.18, 35.67887], [3550.0, 0.08693], [3550.45, 0.04204], [3551.99, 0.00251], [3552.18, 2.87205], [3552.73, 2.19085], [3554.67, 0.002], [3558.81, 25.06818], [3563.37, 2.42406], [3567.91, 2.17808], [3568.01, 0.26174], [3569.44, 18.0678], [3570.0, 0.0562], [3573.37, 2.85786], [3578.91, 0.00785], [3579.51, 3.1901], [3580.67, 3.30976], [3580.82, 60.71906], [3588.18, 2.2893], [3589.0, 0.028], [3591.03, 59.88037], [3600.0, 0.15527], [3600.93, 3.45612], [3601.24, 24.35809], [3603.43, 2.81147], [3604.68, 0.00887], [3611.45, 16.9983], [3611.96, 3.27307], [3621.66, 21.89478], [3624.73, 0.01519]],
 'bids': [[3498.59, 0.12896], [3498.58, 0.10841], [3498.49, 1.01445], [3498.46, 3.08416], [3498.42, 0.97249], [3498.39, 1.32254], [3498.35, 0.76235], [3498.32, 1.15785], [3498.28, 2.18094], [3498.25, 2.3019], [3498.2, 2.17655], [3498.17, 2.06585], [3498.12, 1.23082], [3498.09, 2.35975], [3498.05, 0.76902], [3498.0, 2.42961], [3497.96, 1.0325], [3497.91, 1.72683], [3497.88, 0.96092], [3497.82, 1.15176], [3497.79, 1.00887], [3497.75, 1.57647], [3497.72, 2.38428], [3497.68, 1.45576], [3497.65, 1.6974], [3497.6, 1.96136], [3497.54, 2.99421], [3496.85, 3.0012], [3496.41, 59.81383], [3495.75, 0.04592], [3495.71, 51.69173], [3494.99, 30.18006], [3494.17, 63.15702], [3493.56, 31.38068], [3492.99, 61.32373], [3492.24, 20.16486], [3491.6, 12.00626], [3491.39, 63.00538], [3490.68, 19.92256], [3490.06, 55.39592], [3489.29, 36.73127], [3488.76, 51.29137], [3487.99, 20.22177], [3487.92, 46.89161], [3487.39, 47.7572], [3486.95, 0.05948], [3486.94, 34.6924], [3486.27, 33.61144], [3485.58, 54.2705], [3484.98, 43.76687], [3484.61, 15.00603], [3484.42, 38.78866], [3483.72, 26.57666], [3483.1, 44.64133], [3482.88, 38.57331], [3482.09, 46.8696], [3481.5, 55.72698], [3480.72, 29.57502], [3480.18, 36.1512], [3479.31, 56.96], [3478.63, 23.86597], [3478.1, 66.94343], [3477.25, 42.29285], [3476.65, 58.94984], [3462.28, 0.05777], [3461.27, 55.00379], [3458.72, 2.17137], [3450.57, 60.17071], [3440.02, 18.72912], [3438.66, 3.45415], [3436.51, 2.19008], [3431.14, 3.36178], [3431.04, 3.58268], [3429.46, 62.56879], [3428.0, 0.05834], [3425.66, 3.09903], [3420.0, 0.00611], [3418.72, 20.28883], [3415.16, 2.59092], [3409.97, 2.34787], [3408.34, 19.378], [3400.2, 0.12931], [3400.0, 0.03308], [3394.52, 46.44618], [3394.49, 3.48694], [3394.06, 0.05893], [3392.61, 3.37806], [3392.0, 0.26746], [3385.26, 3.58639], [3384.23, 40.46924], [3379.76, 3.5665], [3378.53, 0.01743], [3374.62, 3.12173], [3373.85, 48.53087], [3370.0, 0.014], [3364.0, 3.61344], [3363.38, 25.8422], [3360.46, 0.05952], [3353.14, 63.6102]],
 'datetime': '2024-03-04T11:34:28.777Z',
 'nonce': None,
```
```
Python v3.11.7
CCXT v4.2.58
bitmart.watchOrderBookForSymbols(['BTC/USDT:USDT', 'ETH/USDT:USDT'])
{'asks': [[65060.3, 8525.0], [65060.4, 10526.0], [65060.5, 6003.0], [65060.6, 4776.0], [65060.7, 3843.0], [65060.8, 850.0], [65060.9, 2398.0], [65061.0, 3188.0], [65066.3, 9138.0], [65066.7, 3411.0], [65066.8, 3458.0], [65066.9, 1519.0], [65067.0, 284.0], [65070.7, 2694.0], [65070.8, 1383.0], [65070.9, 999.0], [65071.0, 2324.0], [65071.3, 3842.0], [65071.4, 384.0], [65071.5, 2075.0], [65071.6, 2459.0], [65071.7, 461.0], [65072.3, 8630.0], [65072.5, 192.0], [65072.6, 312.0], [65074.6, 615.0], [65074.7, 2536.0], [65074.8, 461.0], [65075.5, 11231.0], [65077.2, 8889.0], [65079.0, 5877.0], [65079.1, 1460.0], [65079.2, 1613.0], [65079.3, 3073.0], [65084.1, 6963.0], [65084.7, 7027.0], [65087.6, 3764.0], [65087.7, 1229.0], [65087.8, 77.0], [65087.9, 2804.0], [65088.0, 3859.0], [65088.1, 2351.0], [65088.2, 101.0], [65088.3, 482.0], [65090.1, 14559.0], [65091.0, 555.0], [65093.8, 7194.0], [65095.2, 10589.0], [65097.5, 16773.0], [65099.7, 26.0]],
 'bids': [[65059.8, 7685.0], [65059.7, 5841.0], [65059.6, 6425.0], [65059.5, 8032.0], [65059.4, 9722.0], [65059.3, 8218.0], [65059.2, 7979.0], [65059.1, 9530.0], [65057.0, 11676.0], [65056.4, 10658.0], [65053.0, 11832.0], [65052.8, 9609.0], [65030.8, 8263.0], [65029.0, 7978.0], [65026.3, 7987.0], [65025.4, 20.0], [65021.1, 26.0], [65020.6, 10926.0], [65018.6, 16132.0], [65009.0, 420.0], [64969.4, 423.0], [64951.5, 11890.0], [64928.5, 1.0], [64927.6, 7.0], [64898.1, 18800.0], [64876.2, 15.0], [64869.4, 445.0], [64848.3, 10737.0], [64848.1, 11862.0], [64809.0, 1.0], [64804.7, 470.0], [64800.0, 395.0], [64790.0, 1.0], [64777.4, 12.0], [64776.2, 24692.0], [64749.6, 8322.0], [64600.3, 50047.0], [64569.5, 17007.0], [64537.9, 45.0], [64500.0, 11.0], [64499.7, 20.0], [64431.8, 31.0], [64320.0, 108.0], [64315.3, 21799.0], [64206.0, 1.0], [64180.8, 31.0], [64162.9, 30705.0], [64111.0, 1.0], [64000.0, 7.0], [63900.0, 17.0]],
 'datetime': '2024-03-04T11:35:12.608Z',
 'nonce': None,
 'symbol': 'BTC/USDT:USDT',
```

**Pull request merged**